statistics/handle/lock_stats_handler.go

```diff
@@ -19,20 +19,36 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/pingcap/errors"
 	"github.com/pingcap/tidb/kv"
 	"github.com/pingcap/tidb/parser/ast"
-	"github.com/pingcap/tidb/statistics/handle/cache"
 	"github.com/pingcap/tidb/util/logutil"
 	"github.com/pingcap/tidb/util/sqlexec"
 	"go.uber.org/zap"
 )
 
-// Stats logger.
-var statsLogger = logutil.BgLogger().With(zap.String("category", "stats"))
+const (
+	lockAction     = "locking"
+	unlockAction   = "unlocking"
+	lockedStatus   = "locked"
+	unlockedStatus = "unlocked"
+)
+
+var (
+	// maxChunkSize is the max chunk size for load locked tables.
+	// We use 1024 as the default value, which is the same as the default value of session.maxChunkSize.
+	// The reason why we don't use session.maxChunkSize is that we don't want to introduce a new dependency.
+	// See: https://github.com/pingcap/tidb/pull/46478#discussion_r1308786474
+	maxChunkSize = 1024
```

Comment:
I think we can create a new package in the ```util``` to put all const magic number.

Reply:
This is only for the lock and unlock stats feature.